apps/native-component-list/src/screens/AV/VideoPlayer.tsx

```diff
@@ -144,14 +143,14 @@ function ResizeModeSegmentedControl({
         flex: 1,
       }}>
       <SegmentedControl
-        values={data}
+        values={Object.keys(resizeMap)}
         fontStyle={{ color: Colors.tintColor }}
         selectedIndex={index}
         tintColor={'white'}
         onChange={event => {
           setIndex(event.nativeEvent.selectedSegmentIndex);
         }}
-        onValueChange={value => onValueChange(resizeMap[value])}
+        onValueChange={value => onValueChange(resizeMap[value as keyof typeof resizeMap])}
```

Comment:
```suggestion
        onValueChange={value => onValueChange(resizeMap[value])}
```
Given the change from the other suggestion regarding this Component, it should work like this. _Assuming `SegmentedControl` is well typed_